app/uk/gov/hmrc/economiccrimelevyregistration/forms/mappings/Constraints.scala

```diff
@@ -94,14 +94,13 @@ trait Constraints {
         Valid
     }
 
-  protected def nonEmptySet(errorKey: String): Constraint[Set[_]] =
+  protected def nonEmptySet(errorKey: String): Constraint[Set[_]]                                       =
```

Comment:
Please undo this iff all it is is adding white space.

Reply:
It's added by scalafmtAll command.